pandas/tests/indexing/multiindex/test_sorted.py

```diff
@@ -86,6 +88,25 @@ def test_sort_values_key(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_sort_values_with_na(self):
+        arrays = [
+            array([2, 1, NA], dtype="Int64"),
+            array([1, 2, 3], dtype="Int64"),
+        ]
+        index = MultiIndex.from_arrays(arrays)
+        index = index.sort_values()
+        result = DataFrame(range(3), index=index)
+
+        arrays = [
+            array([1, 2, NA], dtype="Int64"),
+            array([2, 1, 3], dtype="Int64"),
+        ]
+        index = MultiIndex.from_arrays(arrays)
+        index = index.sort_values()
```

Comment:
Could we build `expected` such that `sort_values` isn't called (just in case there's a bug that would affect both calls)?

Reply:
Yes, that was a mistake. Removed the call to sort_values.